lib/linux_admin/disk.rb

```diff
@@ -13,23 +13,27 @@ class Disk
     def str_to_bytes(val, unit)
       case unit
       when 'K' then
-        val.to_f.kilobytes
+        val.to_f * 1_024 # 1.kilobytes
       when 'M' then
-        val.to_f.megabytes
+        val.to_f * 1_048_576 # 1.megabyte
       when 'G' then
-        val.to_f.gigabytes
+        val.to_f * 1_073_741_824 # 1.gigabytes
```

Comment:
So, *technically* a kilobyte is 1000 bytes right? https://en.wikipedia.org/wiki/Kilobyte has a decent chart, but which form is being used is usually ambiguous. I'm assuming you checked out that these were what `1.kilobytes` and such actually resolved to?